doc/DeveloperGuide.md

```diff
@@ -88,6 +88,26 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+**MSS**
+
+1. User requests to rename tag
+2. AddressBook shows a list of persons with the tag
+3. AddressBook requests for confirmation
```

Comment:
The Use Case seeks to model the interaction between an actor and the system.
Therefore the interaction should be alternating between user and the system, user -> system -> user -> system -> etc
In this case, do you know which part should be modified? =)